src/components/CreateAccount.jsx

```diff
@@ -1,9 +1,287 @@
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const navigate = useNavigate();
+
+  const onSubmit = async (data) => {
+    console.log(data);
+
+    let config = {
+        method: "post",
+        maxBodyLength: Infinity,
+        url: "http://192.168.1.34:8484/v1/signup",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        mode: "no-mode",
+        referrerPolicy: "no-referrer",
+        data: data,
+      };
+
+    await axios
+    .request(config)
+    .then((response) => {
+      console.log(response);
+      let accStatus = window.confirm("Account Created Successfully !");
+      if(accStatus){
+          navigate("/userlogin");
+      }
+    })
+    .catch((error) => {
+      console.log(error.message);
+    });
+  };
+
+  const axios = require("axios");
+  //   let data = JSON.stringify(userAccountData);
+  let data = JSON.stringify({
+    firstName: "sameerbhsdsaiya123",
+    lastName: "sexybsdsdoy",
+    gender: "male",
+    email: "samees1211198r.srivastava@fiftyfivetech.in",
+    password: "1660128226",
+  });
+
+  
+
+  
+
   return (
-    <div>CreateAccount</div>
-  )
-}
+    <Container
+      maxWidth="lg"
+      sx={{
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        height: "100vh",
+      }}
+    >
+      <Grid container>
+        <Grid
+          item
+          lg={5}
+          md={5}
+          sm={0}
+          xs={0}
+          sx={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <div
+            style={{
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
+            }}
+          >
+            <img
+              src={CreateAccountImage}
+              alt="signup"
+              width={"120%"}
+              id="loginPageImage"
+            />
+          </div>
+        </Grid>
+
+        <Grid
+          item
+          lg={7}
+          md={7}
+          sm={12}
+          xs={12}
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <div
+            style={{
+              width: "80%",
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
+            }}
+          >
+            <div style={{ width: "35px", height: "35px", margin: "0px 10px" }}>
+              <img
+                src={BrandLogo}
+                alt="55 Technologies"
+                style={{ width: "100%", height: "100%" }}
+              />
+            </div>
+            <h1>Chit-Chat</h1>
+          </div>
+          <div>
+            <h3 style={{ textAlign: "center", margin: "10px 0px" }}>
+              Create a new account
```

Comment:
never put direct static values, store it in some constantfile and then import it from there